.github/ISSUE_TEMPLATE/question.md

```diff
@@ -5,25 +5,10 @@ about: If you have questions, visit GitHub Discussions and Wiki pages
 
 # 💬 Questions #
```

Comment:
I think that rather than having a `question.md` template, we should add a `config.yml` to modify the Issue Selector like so:

```yaml
---
# When we are comfortable with our templates, this should be changed to false
# to force new issues into an existing template.
blank_issues_enabled: true
contact_links:
  - name: Questions
    url: https://github.com/cisagov/.github/QUESTIONS.md
    about: Please also see our documentation and Discussion boards for common questions and topics.
```

@dav3r @jsf9k I would appreciate your thoughts as well. You can see a sample of what it would look like in one of my testing repos [here](https://github.com/mcdonnnj/github-actions-testing/issues/new/choose).

Reply:
I do this... take a look at:
https://github.com/felddy/foundryvtt-docker/blob/develop/.github/ISSUE_TEMPLATE/config.yml

Reply:
I like what @mcdonnnj and @felddy are suggesting.

Reply:
I like the link to direct questions to the repo's GitHub Discussions, as @felddy has shown.

Reply:
Looks good! We can use that yml approach to link directly to Discussions in certain repositories and to QUESTIONS.md for the default/generic. Unless there's a way to have it relatively-refer to the current repository's Discussions.

Reply:
Implemented in https://github.com/cisagov/.github/pull/11/commits/402cf85cbbcc0a0f0073aa572aa4face1adb7e60

Reply:
> Looks good! We can use that yml approach to link directly to Discussions in certain repositories and to QUESTIONS.md for the default/generic. Unless there's a way to have it relatively-refer to the current repository's Discussions.

Would a relative link [like this](../issues) work?

Reply:
> > Looks good! We can use that yml approach to link directly to Discussions in certain repositories and to QUESTIONS.md for the default/generic. Unless there's a way to have it relatively-refer to the current repository's Discussions.
> 
> Would a relative link [like this](../issues) work?

Already merged :cry:

Reply:
The problem I see with that is that not all repositories have Discussions or Wiki pages existent or active.

Reply:
OK

Reply:
> 
> 
> > Looks good! We can use that yml approach to link directly to Discussions in certain repositories and to QUESTIONS.md for the default/generic. Unless there's a way to have it relatively-refer to the current repository's Discussions.
> 
> Would a relative link [like this](../issues) work?

Remember that adding *anything* to `ISSUE_TEMPLATE` in a repository will break the automatic inclusion of these defaults. I had to manually add these issue templates to [cisagov/cyhy_amis](https://github.com/cisagov/cyhy_amis) in https://github.com/cisagov/cyhy_amis/pull/300 because of the monthly redeploy template we added.

I did not have success when testing a relative link in `config.yml` when I tried.